mappings/net/minecraft/client/particle/ParticleFlame.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bnb
+CLASS none/bnb net/minecraft/client/particle/ParticleFlame
```

Comment:
Would prefer `Fire`, personally. Is there a specific reason for `Flame`, just out of curiosity?